cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -37,16 +37,24 @@
         <div class="content-l_col
                     content-l_col-1-3">
             <div class="o-form_group">
-                <fieldset class="o-form_fieldset">
-                    <legend class="a-legend">
+                <div class="m-form-field">
+                    <label class="a-label a-label__heading"
+                            for="{{ form.categories.id_for_label }}">
```

Comment:
TRIVIAL: you aligned the `class` and `for` attributes in https://github.com/cfpb/consumerfinance.gov/pull/6494/files#diff-f17634aa56ba7a2709bc6f1802bdec486df71171809683ad1ed6d12f794a6402R63